cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/AddCommand.java

```diff
@@ -120,8 +120,8 @@ public void execute() {
         Path projectPath = ProjectDirs.findProjectRoot(userDir);
         if (null == projectPath) {
             CommandUtil.printError(errStream,
-                    "not a ballerina project (or any parent up to mount point)\n" +
-                            "You should run this command inside a ballerina project", null, false);
+                    "This is not a Ballerina project (or any parent up to the mount point).\n" +
```

Comment:
Shall we get this clarified with the team and rephrase? I don't really get the message within the parenthesis.

Reply:
Changed to "current directory is not a Ballerina project.\"